src/Nethermind/Nethermind.Consensus/Validators/BlockValidator.cs

```diff
@@ -325,6 +328,46 @@ private bool ValidateDeposits(Block block, IReleaseSpec spec, out string? error)
         return true;
     }
 
+    public bool ValidateValidatorExits(Block block, out string error) =>
+        ValidateValidatorExits(block, _specProvider.GetSpec(block.Header), out error);
+
+
+    private bool ValidateValidatorExits(Block block, IReleaseSpec spec, out string error)
```

Comment:
(Cosmetic) For all errors in this method why not to use BlockErrorMessages class similarly to withdrawals